.drone.yml

```diff
@@ -234,6 +234,37 @@ steps:
     event:
     - tag
 
+- name: docker-tag
```

Comment:
Lets name this `docker-mirror-tag` or something, to represent that its different.

Reply:
This is done.